index/string.go

```diff
@@ -12,6 +12,10 @@ func String(s string) Key {
 	return []byte(s)
 }
 
+func FromString(s string) (Key, error) {
```

Comment:
I think it makes sense to have also these utility functions here for the `FromString` index field. Not sure what the naming convention should be.
`index.Int64`, `index.Int64FromString`? `index.Int64String`?

Reply:
Added the `*String` variants in the last commit, e.g. `index.Int32String` and so on.

Reply:
> Or should I add it for consistency?

As a personal preference I'd go for `index.StringString` to be consistent, but agree that it is not a nice name :sweat:
No strong opinion on this, in the end both are fine for me.